### PR TITLE
Auth by Bawwa; Assigned to Chenuka

### PR DESCRIPTION
**Pull request by sithumonline**
## Purpose

The purpose of this PR is to fix #<issue-number>

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have performed a self-review of my own code
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: src/api/auth/auth.controller.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import AuthService from "./auth.service";
+import {HTTP_STATUS} from "../../utils/http";
+import logger from "../../utils/logger";
+import passport from "passport";
+
+const adminRegister = async (req, res) => {
+    //logger.info('auth.controller.js createAdmin(): ' + JSON.parse(req.body));
```
Comment (Shehanka) [inline src/api/auth/auth.controller.js:7]: remove this unused comment

File: src/app.js [hunk 2]
```diff
@@ -1,19 +1,36 @@
...
+    saveUninitialized: false,
+}));
+
+app.use(passport.initialize());
+app.use(passport.session());
+passport.use(new LocalStrategy(User.authenticate()));
+passport.serializeUser(User.serializeUser());
+passport.deserializeUser(User.deserializeUser());
+
 //app.use(cors({ origin: true, credentials: true }));
```
Comment (Shehanka) [inline src/app.js:32]: why cors disabled?

**Review by Shehanka — CHANGES_REQUESTED** [review]

**Comment by sithumonline** [comment]
I fixed this issue with `local` but now we have to implement this for `JWT` as we couldn't do auth with `local` (smile in pain github edition)

**Review by Shehanka — APPROVED** [review]
Good job

**Pull request merged**